cares_reinforcement_learning/algorithm/mbrl/DYNA_SAC.py

```diff
@@ -0,0 +1,292 @@
+"""
+Sutton, Richard S. "Dyna, an integrated architecture for learning, planning, and reacting."
+
+This code runs automatic entropy tuning
+"""
+
+import copy
+import logging
+import os
+
+import numpy as np
+import torch
+import torch.nn.functional as F
+
+
```

Comment:
rename - DynaSAC, don't use _ in class names

Reply:
Changed